### DISCUSSION TITLE
taxadept prebuilt linux binaries

### DISCUSSION THREAD

**Discussion by umnyaga** [orbitalquark/textadept#499] [General]
I wanted to use taxadept terminal version on my system (CentOS 7). Downloaded the binaries and when I tried to execute textadept-curses I got following error:
```
/textadept-curses: error while loading shared libraries: libncursesw.so.6: cannot open shared object file: No such file or directory
```
Could taxadept be build with static curses library for the distribution?

**Comment by orbitalquark** [comment]
I suspect it could be, but embedding low-level static libraries normally provided by the system is generally frowned upon and could have unintended and unexpected side effects. As mentioned in the FAQ (https://orbitalquark.github.io/textadept/faq.html), it's hard to provide truly cross-platform binaries, so for some systems, compiling Textadept (https://orbitalquark.github.io/textadept/manual.html#compiling) is the way to go.
